packages/editor/src/components/document-bar/index.js

```diff
@@ -184,11 +187,15 @@ export default function DocumentBar( props ) {
 									? decodeEntities( title )
 									: __( 'No title' ) }
 							</span>
-							{ postTypeLabel && ! props.title && (
-								<span className="editor-document-bar__post-type-label">
-									{ '· ' + decodeEntities( postTypeLabel ) }
-								</span>
-							) }
+							<span className="editor-document-bar__post-type-label">
+								{ `. ${
```

Comment:
Please do not change this to a normal dot. It should be a 'middle dot' character `·`

Reply:
This is now revert to use `·` character.